tests/cases/compute_ubo_and_ssbo_layouts.amber

```diff
@@ -0,0 +1,129 @@
+# Copyright 2018 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[compute shader]
+#version 430
+
+// SSBO compiled as std430 by glslang
+layout(set = 0, binding = 0) buffer block00 {
+  float a[3];   // Offset: 0, array stride:   4
+  int b;        //        12
+  mat3 c;       //        16, matrix stride: 16
+  int d[5];     //        64, array stride:   4
+  float e;      //        84
+} b0;
+
+// UBO compiled as std430 by glslang
```

Comment:
The commit message says ubo is 140 not 430?

Reply:
Right. Updated.